tsconfig.json

```diff
@@ -22,7 +30,6 @@
         "SampleApplication/.vscode",
         "ReactTypings",
         "tools/",
-        "src/test/resources/sampleReactNative022Project/",
-        ".vscode-test"

```

Comment:
I think you still need this line - this dir is created each time after running `npm test`